src/mongodb/scores.controller.ts

```diff
@@ -98,17 +125,12 @@ export class ScoresController {
         if (responseTextELE != ' ') {
           if (vowelSignArr.includes(responseTextELE)) {
             if (isPrevVowel) {
-              if (language !== 'hi') {
```

Comment:
Do we need to do the same routine for original text and response text?  code reuse possibility